beluga/include/beluga/sensor/data/occupancy_grid.hpp

```diff
@@ -181,6 +184,14 @@ class BaseOccupancyGrid2 : public BaseLinearGrid2<Derived> {
              return value_traits.is_occupied(value);
            });
   }
+
+  /// Retrieves grid data using GridStatus enum class (Free, Unknown or Occupied) for obstacles.
+  [[nodiscard]] auto unknown_obstacle_data() const {
```

Comment:
@DPR00 not sold on the the name yet. `unknown_obstacle_data` gives no hint as to what you are getting from it.

Reply:
what about `data_status()`?

Reply:
It's really a ternary view, as opposed to the binary view that `obstacle_data` provides. Maybe we should move in that direction? `binary_data()` and `ternary_data()`? @nahueespinosa WDYT?

Reply:
I don't have great ideas for this. I like `ternary_data()`, maybe that's all we need here?

Otherwise, which flavor of binary data we want to favor? Just one or all three of them?
 - `obstacle_data()` (only occupied data)
 - `free_space_data()` (only free data)
 - `unknown_data()` (only unknown data)